tests/subsys/logging/log_syst/src/mock_backend.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SRC_MOCK_BACKEND_H__
+#define SRC_MOCK_BACKEND_H__
+
+#include <logging/log_backend.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+extern const struct log_backend_api mock_log_backend_api;
+
+struct mock_log_backend_msg {
+	/* Not used. */
+	uint8_t data[32];
```

Comment:
What does this comment means ? If that is not used, what is used in `mock_log_backend` ? just `bool panic;` ?

Reply:
Removed the mock_log_backend_msg and its reference from mock_log_backend struct for now as mock backend is not holding any messages for now. Can add back later if its use is identified.